repo/fsrepo/lock/lock.go

```diff
@@ -13,6 +16,10 @@ import (
 // TODO rename repo lock and hide name
 const LockFile = "repo.lock"
 
+func errPerm(path string) error {
+	return fmt.Errorf("failed to take lock at %s: permission denied", path)
```

Comment:
maybe: 
```
"locking %s: permission denied"
```

so that the error prints as:

```
Error: locking <path>: permission denied.
```
(but maybe this is fine)